dist/components/progress-steps/_progress-steps.scss

```diff
@@ -49,24 +53,22 @@ $progress-steps__divider-border-color: hsl(360, 100%, 100%) !default;
         padding-right: 0; // 2
         padding-left: $progress-steps__divider-width + $progress-steps__item-padding; //2
     }
-}
 
 
-// Arrow Shape
-// ---
-//
-// We are using the two (before/after) pseudo elements to create a CSS triangle
-// for our progress step item arrow. We use two on top of each other to create
-// visual separation appearing as either a border or space.
-//
-// The `before` is used as the border or space
-//
-// The `after` is used to create the arrow shape of the `.c-progress-steps__item`
-//
-// 1. The "thickness" of the border/seperator is an illusion created by
-//    this margin-left property
+    // Progress Steps Items: Arrow Shape
```

Comment:
I understand your point. I was also not sure if it should be moved inside the component. But in this case we have a lot of styles and a huge comment what is reasonable to keep out. I am worry though if it's clear why we have 2 `.c-progress-steps__item` in the code.